tools/tpm2_ptool.py

```diff
@@ -949,7 +999,7 @@ def do_token_init(db, path, args):
                 encsobjauth = AESCipher(wrappingobjauth['rhash']).encrypt(sobjauth)
 
             objattrs="restricted|decrypt|fixedtpm|fixedparent|sensitivedataorigin|userwithauth"
-            sobjpriv, sobjpub, sobjpubdata = tpm2.create(pobject['handle'], pobjauthhash, sobjauth, objattrs=objattrs, alg='aes{}'.format(sym_size))
+            sobjpriv, sobjpub, sobjpubdata = tpm2.create(pobject['handle'], pobjauthhash, sobjauth, objattrs=objattrs, alg='rsa2048')
```

Comment:
Isn't rsa2048 the default?

Reply:
I'm not sure, will have to check the code.